.halcyon/sandbox-extra-apps

```diff
@@ -0,0 +1,3 @@
+alex-3.1.3
+happy-1.19.4
+yesod-bin-1.4.3.1
```

Comment:
You should also add [constraints for the sandbox extra apps](https://halcyon.sh/reference/#halcyon_sandbox_extra_apps_constraints).  For example:
- https://github.com/mietek/hl/blob/master/.halcyon/sandbox-extra-apps-constraints/alex-3.1.4.constraints
- https://github.com/mietek/hl/blob/master/.halcyon/sandbox-extra-apps-constraints/happy-1.19.5.constraints